stories/remark-component.stories.tsx

```diff
@@ -1,5 +1,4 @@
 import React from 'react';
-import { text } from '@storybook/addon-knobs';
```

Comment:
migrated to storybook args
https://storybook.js.org/docs/react/writing-stories/args